src/Microsoft.AspNetCore.Antiforgery/Internal/DefaultAntiforgeryTokenStore.cs

```diff
@@ -80,6 +84,8 @@ public void SaveCookieToken(HttpContext httpContext, string token)
             }
             SetCookiePath(httpContext, options);
 
+            _options.ConfigureCookieOptions?.Invoke(options);
```

Comment:
Consistency is good, but folks are already using the old way. I assume you don't want to remove the existing path, just add a way to set it here to?